user-guide/Advanced_Modules/EPM/Tutorials/EPM_Introduction_Tutorial.md

```diff
@@ -0,0 +1,192 @@
+
+- [Step 1: Deploy the DataMiner EPM Integration Training package from the Catalog](#step-1-deploy-the-dataminer-epm-integration-training-package-from-the-catalog)
+- [Step 2: Create a new collector element and rebalance the system](#step-2-create-a-new-collector-element-and-rebalance-the-system)
+- [Step 3: Use EPM relations to filter out alarms](#step-3-use-epm-relations-to-filter-out-alarms)
+- [Step 4: Using the EPM Feed in Dashboards and Low-Code Apps](#step-4-using-the-epm-feed-in-dashboards-and-low-code-apps)
+- [Step 5: Link EPM entities to Views](#step-5-link-epm-entities-to-views)
+
+## Step 1: Deploy the *DataMiner EPM Integration Training* package from the Catalog
+
+1. Go to <https://catalog.dataminer.services/details/b661f936-d6e7-447c-baee-f0a5503e75b4>.
+
+1. Deploy the Catalog item to your DataMiner Agent by clicking the *Deploy* button.
+
+   > [!TIP]
+   > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
+
+1. Open DataMiner Cube and check whether the following items have been added to your DataMiner Agent:
+
+   - A **view** named *EPM Training*.
+
+   - A **front-end element** named *EPM Training FE*.
+
+   - Two **back-end elements** named *EPM Training BE 1* and *EPM Training BE 2*.
+
+   - Three **collector elements** named *EPM Training Collector 1*, *EPM Training Collector 2*, and *EPM Training Collector 3*.
+
+   - The **Topology app** in between the Surveyor and Activity module.
+
+   ![Cube sidebar with Topology app](~/user-guide/images/EPM_GS_step_1.png)
+
+> [!NOTE]
+> If the Topology app does not show up right away, restart the Cube client.
+
+## Step 2: Create a new collector element and rebalance the system
+
+1. Create a new collector element in the *EPM Training* view:
+
+   1. Open the Surveyor in DataMiner Cube.
+
+   1. Right-click the *EPM Training* view (e.g. the root view), and select *New* > *Element*.
+
+   1. Enter the element name *EPM Training Collector 4*, select the DMA where the element should be added, select the *Skyline EPM Integration Training Collector* protocol, and click *Create*.
+
+      > [!NOTE]
+      > If you are using a DMS consisting of multiple DMAs, select the same DMA as the one hosting the *EPM Training BE 2* element. If you do not know which DMA is hosting this element, right-click this element and select *Properties*. The DMA ID is the first number in the ID, before the slash.
+
+   ![Creation of new Collector element](~/user-guide/images/EPM_GS_step_2_1.png)
+
+1. Right-click the newly created element, select *Properties*, and copy the DMA ID/element ID of the element.
+
+   ![DMA ID/element ID in element properties](~/user-guide/images/EPM_GS_step_2_2.png)
+
+1. Configure the *EPM Training BE 2* element as the parent manager of the *EPM Training Collector 4* element:
+
+   1. Open the *EPM Training BE 2* (1) element, and go to the *Configuration* visual page (2) and then the *Collectors* tab (3).
```

Comment:
I've installed version 1.0.1-CU3 of the EPM Integration Training package on my DMA (running 10.4.8) to test this. However, I don't see this Collectors tab on the configuration visual page:
![image](https://github.com/user-attachments/assets/c3a049c3-e69e-4549-ae87-a86f86956c31)
Any idea what went wrong?

Reply:
Not sure, just installed on my local as well (running 10.4.8-) and the pages are showing up. Also inspected the visio and the pages are there. Maybe it's a visual bug? Can you close and open the cube window? Also, did the alarm templates get applied to the FE and BE elements?

Reply:
As it seems to be a bug that's specifically related to my system, we've found a way around this by using a different client PC.